Makefile

```diff
@@ -156,6 +158,11 @@ test/common:
 	git checkout --force FETCH_HEAD -- test/common
 	git reset test/common
 
+dist/patternfly-4-base.css: $(srcdir)/lib/patternfly-4-cockpit.scss
+	$(SASS) $< $@.tmp --source-map --source-map-urls absolute --style compressed --load-path node_modules && \
```

Comment:
When I run `make` in a clean tree, it fails with
```
/bin/sh: ./node_modules/.bin/sass: No such file or directory
```
 Please add `$(WEBPACK_TEST)` as another (second!) dependency to that rule, to make sure that this runs after npm install.

Also, this somehow needs to specify the name of the .map file -- right now it builds `patternfly-4-base.css.tmp.map`. If that's not possible, then let's drop the `.tmp` again (but that would reintroduce the race condition again)

Reply:
It's not possible to specify the source map name. It's rare that we make changed to all these PF4 files so lets not bother so much about it, what do you think?

Reply:
Ack, then let's drop the .tmp stuff again.